onnx/defs/schema.h

```diff
@@ -954,7 +954,7 @@ class OpSchemaRegistry final : public ISchemaRegistry {
 
   class OpSchemaRegisterOnce final {
    public:
-    OpSchemaRegisterOnce(OpSchema& op_schema) {
+    OpSchemaRegisterOnce(OpSchema& op_schema, int max_version=0) {
```

Comment:
should max_version be renames to opset_version_to_load? max_version to me seems unintuitive... max_version gives a false impression thart all schemas till a max_version should be loaded but this is not true right... you only want to load schemas valid for a particular opset version

Reply:
Agreed. I was quite confused about `max_version` and `opset_version_to_load` looks better to me. Also, I will add some comments for it to make it clear.